eng/common/scripts/Helpers/ApiView-Helpers.ps1

```diff
@@ -42,6 +42,11 @@ function Check-ApiReviewStatus($packageName, $packageVersion, $language, $url, $
     {
       Write-Host "API Review is approved for package $($packageName)"
     }
+    else if ($response.StatusCode -eq '202')
+    {
+      Write-Host "Package name $($packageName) is not yet approved by an SDK API approver. Package name must be approved to release a preview version if $($packageName) was never released as GA version."
+      Write-Host "You can check http://aka.ms/azsdk/engsys/apireview/faq for more details on package name Approval."
+    }
     else
```

Comment:
Just to be cautious I would add an if StatusCode -eq 201 here and have a fallback else that represents all other failure cases. For example what happens if we get another error code that doesn't cause an exception?